R/net.accessor.R

```diff
@@ -438,6 +438,35 @@ get_control <- function(dat, item, override.null.error = FALSE) {
   return(out)
 }
 
+#' @rdname net-accessor
+#' @param network index of network for which to get control
+#' @export
+get_network_control <- function(dat, item, override.null.error = FALSE, network) {
```

Comment:
`network` could be the second argument for consistency with the other functions with a mandatory `network` argument